### PR TITLE
Adding copyrights to java, js and jsx files

### PR DESCRIPTION
**Pull request by maheeka**
Resolves https://github.com/ballerinalang/composer/issues/106

### CODE REVIEW

File: modules/web/styleswitcher.js [hunk 1]
```diff
@@ -98,3 +117,23 @@ window.onunload = () => {
 const cookie = readCookie('style');
 const title = cookie || getPreferredStyleSheet();
 setActiveStyleSheet(title);
+/**
```
Comment (hemikak) [inline modules/web/styleswitcher.js]: Wrong place for the copyright header.

File: modules/web/src/plugins/help/dialogs/about-dialog.jsx [hunk 2]
```diff
@@ -62,7 +81,12 @@ class AboutDialog extends React.Component {
                         >
                             <i className="fw fw-wso2 icon" /> Inc.
                         </a>
-                    </div>
+                    </div>/*
+ * Created on Tue Nov 14 2017
```
Comment (hemikak) [inline modules/web/src/plugins/help/dialogs/about-dialog.jsx]: Invalid here.

File: modules/web/styleswitcher.js (hunk 1 above)
Comment (maheeka) [reply modules/web/styleswitcher.js]: Fixed with https://github.com/ballerinalang/composer/pull/4592

File: modules/web/src/plugins/help/dialogs/about-dialog.jsx (hunk 2 above)
Comment (maheeka) [reply modules/web/src/plugins/help/dialogs/about-dialog.jsx]: Fixed with https://github.com/ballerinalang/composer/pull/4592/files

**Pull request merged**